### DISCUSSION TITLE
matmul for pauli strings

### DISCUSSION THREAD

**Discussion by jwnys** [netket/netket#848] [Ideas]
Unless I'm mistaken, pauli strings cannot be efficiently __matmul__ with each other. This occurs quite often though, at least in what I do. I have some code that basically keeps the actual strings and coefficients and performs an efficient matmul just through string operators. Would this be something worth adding to netket?

**Comment by gcarleo** [comment]
Hi, the only way to do matmul (I think) is to multiply strings one by one and adding them up...(possibly simplifying the identities that enter etc). Yes it would be nice to have it implemented, I agree!

**Comment by gcarleo** [comment]
implemented in #955
